qa/acceptance-tests/src/test/java/io/camunda/it/auth/MappingRuleAuthorizationIT.java

```diff
@@ -86,125 +77,113 @@ class MappingRuleAuthorizationIT {
+        .isEmpty();
   }
 
   @Test
   void deleteMappingRuleShouldReturnForbiddenIfUnauthorized(
-      @Authenticated(RESTRICTED) final CamundaClient camundaClient) {
-    assertThatThrownBy(
-            () -> camundaClient.newDeleteMappingRuleCommand("mappingRule1").send().join())
-        .isInstanceOf(ProblemException.class)
-        .hasMessageContaining("403: 'Forbidden'");
+      @Authenticated(RESTRICTED) final CamundaClient camundaClient,
+      @Authenticated(ADMIN) final CamundaClient adminClient) {
+    // given
+    final var id = createRandomMappingRule(adminClient);
+
+    // when / then
+    assertThat((Future<?>) camundaClient.newDeleteMappingRuleCommand(id).send())
+        .failsWithin(Duration.ofSeconds(5))
+        .withThrowableThat()
+        .withRootCauseInstanceOf(ProblemException.class)
+        .withMessageContaining("403: 'Forbidden'");
   }
 
   @Test
   void getMappingRuleShouldReturnForbiddenIfUnauthorized(
-      @Authenticated(UNAUTHORIZED) final CamundaClient camundaClient) {
-    assertThatThrownBy(() -> camundaClient.newMappingRuleGetRequest("mappingRule1").send().join())
-        .isInstanceOf(ProblemException.class)
-        .hasMessageContaining("403: 'Forbidden'");
+      @Authenticated(UNAUTHORIZED) final CamundaClient camundaClient,
+      @Authenticated(ADMIN) final CamundaClient adminClient) {
+    // given
+    final var id = createRandomMappingRule(adminClient);
+
+    // when / then
+    assertThat((Future<?>) camundaClient.newMappingRuleGetRequest(id).send())
+        .failsWithin(Duration.ofSeconds(5))
+        .withThrowableThat()
+        .withRootCauseInstanceOf(ProblemException.class)
+        .withMessageContaining("403: 'Forbidden'");
   }
 
   @Test
   void getMappingRuleShouldReturnMappingRuleIfAuthorized(
-      @Authenticated(RESTRICTED) final CamundaClient camundaClient) {
-    // when - get the mapping rule (this should not throw an exception)
-    final var mappingRule = camundaClient.newMappingRuleGetRequest("mappingRule1").send().join();
+      @Authenticated(RESTRICTED) final CamundaClient camundaClient,
+      @Authenticated(ADMIN) final CamundaClient adminClient) {
+    // given
+    final var id = createRandomMappingRule(adminClient);
+
+    // when
+    final Future<MappingRule> response = camundaClient.newMappingRuleGetRequest(id).send();
 
     // then
-    assertThat(mappingRule.getMappingRuleId()).isEqualTo("mappingRule1");
-    assertThat(mappingRule.getClaimName()).isEqualTo("test-name");
-    assertThat(mappingRule.getClaimValue()).isEqualTo("test-value");
+    assertThat(response)
+        .succeedsWithin(Duration.ofSeconds(5))
+        .satisfies(mappingRule -> assertThat(mappingRule.getMappingRuleId()).isEqualTo(id));
   }
 
   @Test
   void deleteMappingRuleShouldDeleteMappingRuleIfAuthorized(
       @Authenticated(ADMIN) final CamundaClient camundaClient) throws Exception {
```

Comment:
The throws clause can be removed as this method doesn't throw any checked exception